stablehlo/reference/Element.cpp

```diff
@@ -134,6 +147,15 @@ APInt Element::getIntegerValue() const {
   return std::get<APInt>(value_);
 }
 
+bool Element::getBooleanValue() const {
+  if (!isSupportedBooleanType(type_))
+    report_fatal_error(StringRef("Accessing value of a type different than "
+                                 "what is stored in Element object") +
+                       LLVM_PRETTY_FUNCTION);
```

Comment:
I don't think that `LLVM_PRETTY_FUNCTION` is pulling its weight in the interpreter (not just in this PR but everywhere else too). Instead of a oneliner `report_fatal_error("foo")`, we get multiple lines of boilerplate, and it doesn't look like it's worth the cost.

If the idea is to make error messages easily traceable back to the corresponding functions, then we can simply make them unique, right? E.g. "Element is not a boolean" instead of "Accessing value of a type different than what is stored in Element object".

Same applies for other uses of `LLVM_PRETTY_FUNCTION`. E.g. "float & float is unsupported" instead of "Unsupported floating-point type", etc.